### PR TITLE
News dragging

### LINKED ISSUE #260: Unable to manually scroll the trending news section in mobile view/devices
Unable to scroll trending news section left/right in the mobile view like carousels 
image

### PR DESCRIPTION
**Pull request by ravverma**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #260

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/research/equity
- After: https://news-dragging--icicidirect--aemsites.hlx.live/research/equity

### CODE REVIEW

File: blocks/trending-news/trending-news.js [hunk 1]
```diff
@@ -98,78 +95,43 @@ function createNewsCards(item) {
...
-        while (index < cards.length) {
-          if (index === currentIndex) {
-            cards[index].style.opacity = 1;
-          } else {
-            cards[index].style.opacity = 0;
-          }
-          index += 1;
-        }
-      }
+  let forward = true;
```
Comment (ravverma) [inline blocks/trending-news/trending-news.js:125]: took this as opportunity to refactor the code

**Comment by vivesing** [comment]
Few observations,
1. Hide the horizontal scroll bar, visible when the user is scrolling manually
2. When the user has scrolled to a position, stop the auto-scroll action, this leads to scrolled items getting changed.

**Review by sdmcraft — APPROVED** [review]
Please see why PSI check is failing. Rest lgtm.

**Comment by ravverma** [comment]
> Please see why PSI check is failing. Rest lgtm.

re-run cleared PSI

**Comment by ravverma** [comment]
> Few observations,
> 
> 1. Hide the horizontal scroll bar, visible when the user is scrolling manually
> 2. When the user has scrolled to a position, stop the auto-scroll action, this leads to scrolled items getting changed.

fixed

**Review by vivesing — APPROVED** [review]

**Pull request merged**